nilearn/__init__.py

```diff
@@ -9,66 +9,21 @@
 See http://nilearn.github.com for complete documentation.
 """
 
-from version import __version__
+from .version import _check_module_dependencies, __version__
 
 
-def _import_module_with_version_check(module_name, minimum_version):
-    """Check that module is installed with a recent enough version
-    """
-    from distutils.version import LooseVersion
+# Each time nilearn is imported, check module dependencies
```

Comment:
OK.  The behavior of `_check_module_dependencies` was unclear to me (whether it warns or throws), but I like the idea of having really clear goals with comments (vs. good naming of functions).  Removed.